app/apphandlers/setup.go

```diff
@@ -178,9 +180,10 @@ func monitorDestRouters(ctx context.Context, routerFactory *router.Factory, batc
 	routerFactory.RouterDB.DeleteExecuting()
 	batchRouterFactory.RouterDB.DeleteExecuting()
 
-	for config := range ch {
-		sources := config.Data.(backendconfig.ConfigT)
+	for workspaceConfig := range ch {
+		sources := workspaceConfig.Data.(backendconfig.ConfigT)
 		enabledDestinations := make(map[string]bool)
+		isolateFlagMap := make(map[string]bool)
```

Comment:
this should be defined outside the range loop, correct?

Reply:
Yeah I should do that

Reply:
It is outside the range loop right?